### PR TITLE
[#45] Create one runtime distribution per CSP plugin (HuaweiCloud & Openstack for now)

### PR DESCRIPTION
**Pull request by jbonofre**

### CODE REVIEW

File: runtime/src/main/resources/osc-huaweicloud.json [hunk 1]
```diff
@@ -1,14 +1,15 @@
 {
-  "name": "open-services-cloud",
+  "name": "osc-hc",
```
Comment (niuzhenguo) [inline runtime/src/main/resources/osc-huaweicloud.json:2]: how about using `osc-huaweicloud` instead of `osc-hc`, to make it more recognizable.

Comment (jbonofre) [reply runtime/src/main/resources/osc-huaweicloud.json:2]: Good idea. I wanted something "short", but I agree. Let me update to use full qualified name ;)

**Review by niuzhenguo — APPROVED** [review]
Thanks!

File: runtime/src/main/resources/osc-huaweicloud.json (hunk 1 above)
Comment (niuzhenguo) [reply runtime/src/main/resources/osc-huaweicloud.json]: not sure if `hc` can be a short name for huaweicloud :P

**Review by iskey — APPROVED** [review]
/lgtm

**Pull request merged**